python/servo/post_build_commands.py

```diff
@@ -95,8 +95,18 @@ def run(self, params, release=False, dev=False, android=False, debug=False, debu
                 print("Could not find a suitable debugger in your PATH.")
                 return 1
 
+            command = self.debuggerInfo.path
+            if debugger == 'gdb' or debugger == 'lldb':
+                rustCommand = 'rust-' + debugger
+                try:
+                    result = subprocess.call([rustCommand, '--version'], env=env, stdout=open(os.devnull, 'w'))
```

Comment:
I would probably do subprocess.check_output(['which', rustCommand]) (if you do that, you should catch CalledProcessError)